source/includes/api-reference/_api-reference.html.md

```diff
@@ -229,6 +254,153 @@ This message has no extra data.
 #### TRAINING_INITIALIZED
 This message has no extra data.
 
+
+# BRAIN Metrics
+
+The metrics endpoints will return data from the specified version of a BRAIN's
+training. They can be useful for analyzing what happened during training.
+
+## Episode Value
+
+Episode value contains data about each training episode for a given version of a BRAIN.
+This represents what the AI has been rewarded and what it's currently learning
+through simulation.
+
+> Request
+
+```text
+GET /v1/{userName}/{brainName}/{version}/metrics/episode_value
+```
+
+| Parameter | Description |
+| --- | --- |
+| userName | Name of the user who has the BRAIN |
+| brainName | Name of the BRAIN |
+| version | Version of the BRAIN |
+
+> Example JSON Response
+
+```json
+[
+  {
+    "episode": 1,
+    "lesson": "balancing",
+    "value": 25,
+    "concept": "balance",
+    "time": "2017-11-10T06:37:11.712068096Z"
+  },
+  {
+    "episode": 2,
+    "lesson": "balancing",
+    "value": 43,
+    "concept": "balance",
+    "time": "2017-11-10T06:37:11.739321856Z"
+  },
+  {
+    "episode": 3,
+    "lesson": "balancing",
+    "value": 16,
+    "concept": "balance",
+    "time": "2017-11-10T06:37:11.769031936Z"
+  }
+]
+```
+
+### Response
+
+| Parameter | Description |
+| --- | --- |
+| value | The episode's cumulative reward value |
+| episode | The episode for this value |
+| iteration | Total number of simulator iterations that have occurred |
+| time | Timestamp for when this value occurred |
+| concept| The concept this value corresponds to |
+| lesson | The lesson this value corresponds to |
+
+## Test Pass Value
+
+Test pass value contains data for test passes that occur once every 20 episodes
+during training for a given version of a BRAIN.
+This represents the AI's highest reward value achieved at a given time.
+
+> Request
+
+```text
+GET /v1/{userName}/{brainName}/{version}/metrics/test_pass_value
+```
+
+| Parameter | Description |
+| --- | --- |
+| userName | Name of the user who has the BRAIN |
+| brainName | Name of the BRAIN |
+| version | Version of the BRAIN |
+
+> Example JSON Response
+
+```json
+   Example needed
```

Comment:
I couldn't get to alpha to test this. Could you please add a sample here?

Reply:
I just copied the one from `episode_value` since they have the same, if you think it's more correct to have them be a little different, I can fluke some value or something